parmed/tools/add1264.py

```diff
@@ -157,6 +157,17 @@ def params1264(parm, mask, c4file, watermodel, polfile, tunfactor):
                     # There are two C4 terms need to add together between two
                     # different ions
                     result[idx] += c4 / WATER_POL * pol * tunfactor
+
+    # Print the C4 parameters
+    print('Print the C4 parameters (shown with only one decimal place) : ')
+    for i in range(0, ntypes):
```

Comment:
The 0 in range is redundant, so you can replace this with just range(ntypes). This is what is done everywhere else in ParmEd.